specification/sql/resource-manager/Microsoft.Sql/preview/2020-08-01-preview/Databases.json

```diff
@@ -468,10 +464,25 @@
           {
             "name": "databaseName",
             "in": "path",
-            "description": "The name of the database to be paused.",
+            "description": "The name of the database to failover.",
             "required": true,
             "type": "string"
           },
+          {
+            "name": "replicaType",
```

Comment:
Can you please make sure to add an example (or use the existing one) with this query string as well?

Reply:
Probably? I didn't know what algorithm git uses but I noticed that has different ones like patience, anyways, just checkout your PR and noticed that the examples are correct.

Reply:
LOL~ algorithm sometimes cannot works